packages/teleterm/src/ui/StatusBar/useActiveDocumentClusterBreadcrumbs.ts

```diff
@@ -31,6 +31,6 @@ export function useActiveDocumentClusterBreadcrumbs(): string {
 
```

Comment:
Same as in the previous comment, could you update the definition of `getResourceUri` so that it reads the fields directly?

Reply:
Done